docs/docs/api/runonJS.md

```diff
@@ -0,0 +1,83 @@
+---
+id: runOnJS
+title: runOnJS
+sidebar_label: runOnJS
+---
+
+When you call a function on UI thread you can't be sure if you call a worklet or a callback from JS thread. To make it more transparent we introduce runOnJS method which calls a callback asynchronously. If you call a callback without this method then an exception will be thrown.
+
+#### Note
```

Comment:
I think this should be a quote block (do we have any guidelines on docs structure?)

I'd add a short explanation of why we can't wrap the function directly.

Reply:
Anyway, maybe we should consider wrapping functions automatically?

Reply:
fixed fd4e410